src/main/scala/karazin/scala/users/group/week3/Homework.scala

```diff
@@ -16,35 +16,54 @@ object Homework:
     def toInt: Int
     
     // Optional task
-    def fromInt(int: Int) = ???
-  
+    def fromInt(int: Int) =
+      if (int <= 0) Zero
```

Comment:
I'd throw an exception here, because it's not obvious that `*.fromInt(-1) == Zero`.